src/main/java/org/folio/dataexp/service/export/strategies/HoldingsExportStrategy.java

```diff
@@ -238,7 +242,15 @@ private Map<String, Set<UUID>> getTenantIds(Set<UUID> ids, String centralTenantI
       log.info("ID: {}, tenant: {}, actualTenant: {}", id, curTenant, folioExecutionContext.getTenantId());
       if (nonNull(curTenant)) {
         if (availableTenants.contains(curTenant) || curTenant.equals(centralTenantId)) {
-          idsMap.computeIfAbsent(curTenant, k -> new HashSet<>()).add(id);
+          try {
+            permissionsValidator.checkInstanceViewPermissions(curTenant);
```

Comment:
Check permissions is normal flow, i.e we expect allow or forbidden for operation. Is it possible to use if-statement instead of exception handling because if-statement requires less time for processing.

Reply:
If absence of permission is a normal flow, then ViewPermissionDoesNotExist exception doesn't make sense. Should I remove it?

Reply:
Removed try/catch.